[INFO -- 2017-03-08 00:12:23]
Starting flip_value script
version: 2.0
[INFO -- 2017-03-08 00:12:23]
Initialization strings:
[INFO -- 2017-03-08 00:12:23]
(1) target extended-remote | ssh -T mic0 gdbserver --multi -
[INFO -- 2017-03-08 00:12:23]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-08 00:12:23]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-08 00:12:23]
(4) file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-03-08 00:12:23]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-03-08 00:12:23]
(6) set args -s 2048 -n 228 -i /micNfs/gdb-fault-injection/injector/bin/lud/input_2048_th_228 -g /micNfs/gdb-fault-injection/injector/bin/lud/gold_2048_th_228
[INFO -- 2017-03-08 00:12:23]
initSignal:2
[INFO -- 2017-03-08 00:12:23]
endSignal:4
[INFO -- 2017-03-08 00:12:23]
seqSignal:30
[INFO -- 2017-03-08 00:12:23]
faultMode:3
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[EXCEPTION -- 2017-03-08 00:12:33]
pythonException: invalid frame
[INFO -- 2017-03-08 00:12:33]
Backtrace BEGIN:
#0  0x0000000000404caf in L_lud_omp_123__par_loop2_2_5 () at lud_omp.c:148
#1  0x00007ffff790ba1f in ?? ()
#2  0x00007fffffffd9c0 in ?? ()
#3  0x00007ffff4ea0100 in ?? ()
#4  0x0000000000000001 in ?? ()
#5  0x0000000000000003 in ?? ()
#6  0x0000000000000000 in ?? ()

143	            for (i = 0; i < BS; i++)
144	            {
145	                for (k=0; k < BS; k++) {
146	#pragma omp simd 
147	                    for (j = 0; j < BS; j++) {
148	                        sum[j] += temp_left[BS*i + k] * temp_top[BS*k + j];
149	                    }
150	                }
151	#pragma omp simd 
152	                for (j = 0; j < BS; j++) {

Backtrace END
Memory content before bitflip:0x000x080x000x00
Fault Mode: Zero bit-flip
base address to flip value: 0x7fffffffd9c8
address max offset: 4
Memory content after  bitflip:0x000x000x000x00
frame name: L_lud_omp_123__par_loop2_2_5
symbol name: size
symbol filename: lud_omp.c
symbol line: 123
value: 0
value address: 0x7fffffffd9c8
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2017-03-08 00:12:33]
Fault Injection Successful after 10.7236509323s
[INFO -- 2017-03-08 00:12:58]
event type: exit
[INFO -- 2017-03-08 00:12:58]
exit code: 1
